kubernetes/loculus/templates/_s3-endpoint.tpl

```diff
@@ -0,0 +1,23 @@
+{{- define "loculus.s3Url" -}}
+  {{- if $.Values.runDevelopmentS3 }}
+    {{- if eq $.Values.environment "server" -}}
+        {{- (printf "https://s3%s%s" $.Values.subdomainSeparator $.Values.host) -}}
+    {{- else -}}
+        {{- "http://localhost:8084" -}}
+    {{- end -}}
+  {{- else -}}
+    {{- $.Values.s3.bucket.endpoint }}
+  {{- end -}}
+{{- end -}}
+
+{{- define "loculus.s3UrlInternal" -}}
+  {{- if $.Values.runDevelopmentS3 }}
+    {{- if eq $.Values.environment "server" -}}
+        {{- (printf "https://s3%s%s" $.Values.subdomainSeparator $.Values.host) -}}
```

Comment:
I don't immediately understand why this isn't still `http://loculus-minio-service:8084` - though I'm not saying it's wrong

Reply:
I think you're probably right! Just copy-pasted a bit haphazardly 😬 (either URL should be fine, but we can use the proper internal one 👍 )

Reply:
https://github.com/loculus-project/loculus/pull/4025/commits/f62c2355d2bf632536aa5af215926906ccbbf732